assets/js/components/Dashboard/NewPaMessage/AssociateAlertPage.tsx

```diff
@@ -0,0 +1,339 @@
+                  >
+                    {serviceType}
+                  </Button>
+                );
+              })}
+            </ButtonGroup>
+          </Col>
+          <Col>
+            <AssociateAlertsTable
+              alerts={alerts}
+              messageStateFilter={selectedMessageState}
+              serviceTypeFilter={selectedServiceType}
+              setAssociatedAlert={setAssociatedAlert}
+              setShowAlertModal={setShowAlertModal}
+            />
+          </Col>
+        </Row>
+      </Container>
+      <Modal className="alert-items-modal" show={showAlertModal} centered>
+        <Modal.Header
+          closeButton
+          closeVariant="white"
+          onHide={() => {
+            setAssociatedAlert({} as Alert);
+            setShowAlertModal(false);
+          }}
+        >
+          Select items from alert
+        </Modal.Header>
+        <Modal.Body>
+          <div className="alert-description">
+            <div className="alert-description__header">
+              <div className="alert-description__header-id-label">
+                Alert ID:{" "}
+              </div>
+              <div className="alert-description__header-id">
+                {associatedAlert.id}
+              </div>
+            </div>
+            {associatedAlert.header}
+          </div>
+          <div className="checkbox">
+            <Form.Check
+              id="effect-period-checkbox"
+              className="checkbox"
+              label="End PA/ESS message at the end of alert effect period"
+              checked={endWithEffectPeriod}
+              onChange={() => setEndWithEffectPeriod(!endWithEffectPeriod)}
+            />
+          </div>
+          <div className="checkbox">
+            <Form.Check
+              id="import-locations-checkbox"
+              label="Import locations"
+              checked={importLocations}
+              onChange={() => setImportLocations(!importLocations)}
+            />
+            <div className="subtext">Locations will be editable</div>
+          </div>
+          <div className="checkbox">
+            <Form.Check
+              id="import-message-checkbox"
+              label="Import message"
+              checked={importMessage}
+              onChange={() => {
+                setImportMessage(!importMessage);
+              }}
+            />
+            <div className="subtext">Message will be editable</div>
+          </div>
+        </Modal.Body>
+        <Modal.Footer>
+          <Button
+            onClick={() => {
+              setAssociatedAlert({} as Alert);
+              setShowAlertModal(false);
+            }}
+            className="cancel-button"
+            variant="link"
+          >
+            Cancel
+          </Button>
+          <Button
+            onClick={() => {
+              if (importMessage) {
+                setVisualText(associatedAlert.header);
+                setPhoneticText(associatedAlert.header);
```

Comment:
Because users will still need to review audio and reviewing audio sets the phonetic text, you can leave this out and just set visual.